src/ncp/spinel.h

```diff
@@ -683,6 +688,8 @@ typedef enum
     SPINEL_PROP_NET_KEY_SWITCH_GUARDTIME
                                         = SPINEL_PROP_NET__BEGIN + 10, ///< [L]
 
+    SPINEL_PROP_NET_PSKC                = SPINEL_PROP_NET__BEGIN + 11, ///< [D]
```

Comment:
Can you add some documentation for this to the Spinel document? At least a placeholder section.